src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralBrowseObject.xaml

```diff
@@ -371,7 +371,7 @@
     </StringProperty.DataSource>
   </StringProperty>
 
-  <StringProperty Name="Version"
+  <StringProperty Name="PackageVersion"
```

Comment:
Are we 100% sure it's ok to remove `Version` on `GeneralBrowseObject`?

//cc: @davidwengier @davkean

Reply:
In the future be aware that this is an API change - these properties are available to consumers inside VS.